smoke/README.md

```diff
@@ -0,0 +1,105 @@
+     Then the ninja should run for his life
+</pre>
+Python Step Implementations
+Steps used in the scenarios are implemented in Python files in the “steps” directory. You can call these whatever you like as long as they use the python *.py file extension. You don’t need to tell behave which ones to use - it’ll use all of them.
+
+The full detail of the Python side of behave is in the [API documentation](https://behave.readthedocs.io/en/stable/api.html).
+
+Steps are identified using decorators which match the predicate from the feature file: given, when, then and step (variants with Title case are also available if that’s your preference.) The decorator accepts a string containing the rest of the phrase used in the scenario step it belongs to.
+
+Given a Scenario:
+<pre>
+Scenario: Search for an account
+   Given I search for a valid account
+    Then I will see the account details
+</pre>
+Step code implementing the two steps here might look like (using selenium webdriver and some other helpers):
+<pre>
+@given('I search for a valid account')
+def step_impl(context):
+    context.browser.get('http://localhost:8000/index')
+    form = get_element(context.browser, tag='form')
+    get_element(form, name="msisdn").send_keys('61415551234')
+    form.submit()
+
+@then('I will see the account details')
+def step_impl(context):
+    elements = find_elements(context.browser, id='no-account')
+    eq_(elements, [], 'account not found')
+    h = get_element(context.browser, id='account-head')
+    ok_(h.text.startswith("Account 61415551234"),
+        'Heading %r has wrong text' % h.text)
+</pre>
+Openshift dedicated python modules [openshift.py](https://github.com/openshift/csi-driver-shared-resource/tree/master/smoke/features/steps/openshift.py), [command.py](https://github.com/openshift/csi-driver-shared-resource/tree/master/smoke/features/steps/command.py), [project.py](https://github.com/openshift/csi-driver-shared-resource/tree/master/smoke/features/steps/project.py).These modules are used as a wrapper for openshift cli,one can create the Openshift object & use the functionalities to check the resources.
+<pre>
+└── steps
+    ├── command.py
+    ├── openshift.py
+    ├── project.py
+</pre>
+Visit [Behave](https://behave.readthedocs.io/en/stable/tutorial.html) for more info.
+# Test Results 
+<pre>- The test results are JUnit files generated for each feature & are collected in out dir post test run is complete
+</pre>
+# Contribution Workflow
+<p>Step 1: For each feature QE/Dev to create a feature file inside the smoke/features directory, the feature file name should be synonymus with the feature being developed.</p>
+<p>Step 2: Team to review the feature file and post approval QE to add the feature into polarion.</p>
+<p>Step 3: Once the feature file is approved then the contibutor can go ahead and work on the step definitions for the feature. The step definition file(.py) needs to be in the smoke/features/steps/ directory.</p>
```

Comment:
Use a numbered list instead of html <p> tags
```suggestion
1. For each feature QE/Dev to create a feature file inside the smoke/features directory, the feature file name should be synonymus with the feature being developed.
2. Team to review the feature file and post approval QE to add the feature into polarion.
3. Once the feature file is approved then the contibutor can go ahead and work on the step definitions for the feature. The step definition file(.py) needs to be in the smoke/features/steps/ directory.
```